src/components/ErrorPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { Link, useLocation } from "react-router-dom";
+import getIcon from "../misc/getIcon";
+
+export default function ErrorPage() {
+  const location = useLocation();
+  const [isNoUser] = useState(location.pathname.includes('notFound'));
```

Comment:
Just a question, why do you put the value as a state instead of using a variable? I think `React.useState` is great when you need to manage state hence when you're also using `setState` then passing props to other components.

Can you try this instead?
```
const isNoUser = location.pathname.includes('notFound');
```